### DISCUSSION TITLE
Linear gradient in font color

### DISCUSSION THREAD

**Discussion by HilkopterBob** [zauberzeug/nicegui#651] [Q&A] [answered]
havnt found a solution for setting the  color of a text from a label to a linear gradient.
tried:

```py
ui.label("text").style("color:linear-gradient(to right,red,yellow)").classes("text-h6")
```

in quasars documentation is nothing, besides a forum post that states that this solution _should_ work.

So is this a limitation coming from nicegui or is it an underlying problem in quasar?

**Comment by falkoschindler** [comment, marked as answer]
Interesting question, @HilkopterBob! I think this is a matter of CSS rather than Quasar or NiceGUI.
Following some CSS tutorial about [gradients on text](https://fossheim.io/writing/posts/css-text-gradient/) leads to something like this:

```py
ui.label('Hello world!').style('''
  font-size: 72px;
  background: linear-gradient(to right, red, yellow);
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
''')
```

**Reply by HilkopterBob** [reply]
thats perfect, thank you! 
for future reference:
the colors can be written in hex too!

**Comment by rodja** [comment]
@HilkopterBob that would be a very nice demo in the documentation. We would love if  you could add it? Here are the instructions: https://github.com/zauberzeug/nicegui/blob/main/CONTRIBUTING.md#additional-demos

**Reply by HilkopterBob** [reply]
definitely worth a thought!
